newIDE/app/src/AssetStore/ExtensionStore/InstallExtension.js

```diff
@@ -17,7 +14,7 @@ export const installExtension = async (
   i18n: I18nType,
   project: gdProject,
   eventsFunctionsExtensionsState: EventsFunctionsExtensionsState,
-  extensionShortHeader: ExtensionShortHeader
+  extensionShortHeader: { url: string }
```

Comment:
Could almost have been renamed `extensionUrl` (even if it's an object, which is fine)